jibri/rootfs/etc/cont-init.d/11-get-https-cert

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/with-contenv bash
+
+while ! openssl s_client -showcerts -connect meet.jitsi:443 < /dev/null | openssl x509 -outform PEM > /usr/local/share/ca-certificates/meet.crt; do sleep 1; done
```

Comment:
Tried this PR, but I get errors (certificate in the web container is created correctly):

```
openssl s_client -showcerts -connect meet.jitsi:443 < /dev/null
139985578098752:error:0200206F:system library:connect:Connection refused:../crypto/bio/b_sock2.c:108:
139985578098752:error:2008A067:BIO routines:BIO_connect:connect error:../crypto/bio/b_sock2.c:109:
connect:errno=111
```

I even adjusted the port to my local setup (8443), where I overrode the web port because the docker containers run behind a proxy. Now this somehow means jibri doesn't see the `meet.jitsi` network? But it's included in the network.
Would it not be easier to share a volume between `web` and `jibri` and use that generated .crt directly?

Reply:
If you've purged the config dir, maybe the web container was generating dhparams file, which can take a while before nginx starts.

Reply:
Shared volume would be equally good, I think. In both cases you need the retrying loop (or generating the certificates before running docker-compose). In both cases you need to add the certificate to the database. So it would be slightly simpler (cp instead of openssl), but would need additional Docker volume.

Reply:
Slightly changed that part and manually transfered the certificate now:
```suggestion
while [ ! -f /usr/local/share/ca-certificates/meet.crt ]
do
  sleep 1
  if CERTS=$(openssl s_client -showcerts -connect meet.jitsi:443 < /dev/null); then
    echo "$CERTS" | openssl x509 -outform PEM > /usr/local/share/ca-certificates/meet.crt
  fi
done
```
Result is this:
```
jibri_1     | Updating certificates in /etc/ssl/certs...
jibri_1     | 1 added, 0 removed; done.
jibri_1     | Running hooks in /etc/ca-certificates/update.d...
jibri_1     |
jibri_1     | Adding debian:meet.pem
jibri_1     | done.
jibri_1     | done.
jibri_1     | Fontconfig warning: "/etc/fonts/fonts.conf", line 100: unknown element "blank"
jibri_1     | [0410/173754.459884:INFO:headless_shell.cc(619)] Written to file /tmp/s.png.
jibri_1     | certutil: function failed: SEC_ERROR_BAD_DATABASE: security library: bad database.
jibri_1     | [cont-init.d] 11-get-https-cert: exited 0.
jibri_1     | [cont-init.d] done.
```
:/
After initializing the database with certutil like below this error doesn't come up, but I'm still having issues with connecting to `meet.jitsi:443` from the jibri container. I doubt that is related to this fix, but maybe also a root cause for the issue it (I) tries to solve.